app/views/declaration/submission_confirmation_page.scala.html

```diff
@@ -35,10 +37,10 @@ <h1 class="heading-medium">@messages("supplementary.confirmation.whatHappensNext
 
     <p>
         @messages("supplementary.confirmation.explanation")
-        <a href="/customs-declare-exports/submissions">
+        <a href="@routes.SubmissionsController.displayListOfSubmissions()">
             @messages("supplementary.confirmation.explanation.linkText")
         </a>
     </p>
 
-    @components.button_link(messageKey = "supplementary.confirmation.submitAnotherDeclaration", href = "/customs-declare-exports/choice")
+    @components.button_link(messageKey = "supplementary.confirmation.submitAnotherDeclaration", href=routes.ChoiceController.displayPage().url)
```

Comment:
Maybe button_link should take `Call` instead `href` string

Reply:
Probably, it's used in quite a few places though so I dont want to change it now